### PR TITLE
Add CFN Policy Support

### PR DESCRIPTION
**Pull request by IanSavchenko**
* Fixes:
* Breaking change: [ ]

---

Will need to go into atex-platform and atex-consumer-sov and remove duplicate stuff from there after update.

### CODE REVIEW

File: bin/aws-cloudformation-create-stack-policy [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
```
Comment (andreineculau) [inline bin/aws-cloudformation-create-stack-policy]: i think this file needs to be added to `.editorconfig` as an exception

File: bin/aws-cloudformation-create-stack-policy [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+let _ = require('lodash-firecloud').default;
+let fs = require('fs');
+
+let createStackPolicy = async function({cfnTemplatePath}) {
+  let cfnTemplate = JSON.parse(fs.readFileSync(cfnTemplatePath));
+  let resources = _.keys(cfnTemplate.Resources);
+  let retainedResources = _.filter(resources, function(resourceName) {
+    return _.get(cfnTemplate.Resources[resourceName], 'DeletionPolicy') === 'Retain';
+  });
```
Comment (andreineculau) [inline bin/aws-cloudformation-create-stack-policy]: just nit-picking: doesn't this work if you ignore the "wasted" memory?
`let retainedResources = _.keys(_.pickBy(cfnTemplate.Resources, {DeletionPolicy: 'Retain'}));`

File: repo/cfn/inc.mk [hunk 3]
```diff
@@ -116,6 +124,11 @@ $(CFN_JSON_FILES): %.cfn.json: %/index.js %-setup %.cfn.json/lint ## Generate st
 %.cfn.json/rm: %-setup ## Remove stack.
 	$(ECHO_DO) "Removing $(STACK_NAME) stack..."
 	$(call $(STACK_STEM)-pre-rm)
+	$(ECHO_DO) "Removing $(STACK_NAME) stack policy (allowing all changes)..."
```
Comment (andreineculau) [inline repo/cfn/inc.mk]: maybe execute this before line 125? so that whatever in `pre-rm` can actually be done?

**Comment by andreineculau** [comment]
> Will need to go into atex-platform and atex-consumer-sov and remove duplicate stuff from there after update.

I just did that in all repos (I hope).

@IanSavchenko doesn't this close #107 ?

**Comment by IanSavchenko** [comment]
True. Closed.

**Pull request merged**